### PR TITLE
intial commit for index file 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Please have a look if there is anything you don't like

File: index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('Hello World!')
```
Comment (theharshpat) [inline index.js]: add "hello from 100x dev"

**Comment by ravimaini** [comment]
I just fixed the console line from hello world! to hello from 100x dev

File: index.js (hunk 1 above)
Comment (ghost) [reply index.js]: I just fixed the console line from hello world! to hello from 100x dev

**Comment by ghost** [comment]
Have a look just pushed the latest code.

**Pull request merged**